src/main/java/org/usfirst/frc/team1294/robot/commands/ClimbRope.java

```diff
@@ -0,0 +1,44 @@
+package org.usfirst.frc.team1294.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import org.usfirst.frc.team1294.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class ClimbRope extends Command {
+
+  public ClimbRope() {
+    super("Drive motor");
+    requires(Robot.climbingSubsystem);
+  }
+
+  @Override
+  protected void initialize() {
+
+  }
+
+  @Override
+  protected void execute() {
+    XboxController joystick = Robot.oi.getJoystick();
```

Comment:
A complicated command like driving the robot with a joystick is normally the exception. Simple things like actuating a single motor are best done with assignments in OI. The difference is desired.